src/bin/cargo.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(non_snake_case)]
```

Comment:
Hm, let's `allow` directly on `flag_z` fields perhaps? Alternatively, docopt uses serde now, so perhaps we can have `flag_z` and rename it to `flag_Z` on serde level?